doc/classes/InputEventScreenDrag.xml

```diff
@@ -13,12 +13,22 @@
 		<member name="index" type="int" setter="set_index" getter="get_index" default="0">
 			The drag event index in the case of a multi-drag event.
 		</member>
+		<member name="pen_inverted" type="bool" setter="set_pen_inverted" getter="get_pen_inverted" default="false">
+			Returns [code]true[/code] when using the eraser end of a stylus pen.
+			[b]Note:[/b] This property is implemented on Linux, macOS and Windows.
```

Comment:
I don't see any implementation code in this PR?

Reply:
Removed the comment (copy and paste from `InputEventMouseMotion`), `pen_inverted` is not supported by Apple Pencil (there's no "eraser" end), but I have added it to match `InputEventMouseMotion` properties, since one can be created for another via `Input::set_emulate_touch_from_mouse()` and  `Input::is_emulating_touch_from_mouse()`.